gallery/container-insights/Microsoft.ContainerService-managedClusters.json

```diff
@@ -132,8 +132,15 @@
 					"description": "Provide monitoring for services in Open Service Mesh",
 					"name": "OSM monitoring",
 					"isPreview": true
+				},
+				{
+					"id": "Workbooks/AKS/Syslog",
+					"author": "Microsoft",
+					"description": "Provide summary and visualisations for Syslog",
+					"name": "Syslog",
+					"isPreview": true
```

Comment:
I don't think isPreview is needed here, can be removed

Reply:
Done